pkgs/applications/video/video-trimmer/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, rustPlatform
+, glib
+, gtk4
+, libadwaita
+, blueprint-compiler
+, gst_all_1
+, ffmpeg-full
+, wrapGAppsHook4
+, appstream-glib
+, desktop-file-utils
+}:
+stdenv.mkDerivation rec {
+  pname = "video-trimmer";
+  version = "0.7.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "YaLTeR";
+    repo = "video-trimmer";
+    rev = "v${version}";
+    sha256 = "sha256-D7wjJkdqqjjwwYEUZnNr7hFQK59wfTnaCLXCy+SK8Jo=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-cB5dVrEbISvHrOb87uVZSkT694VKtPtyk+c1tYNCTp0=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    blueprint-compiler
+    wrapGAppsHook4
+    appstream-glib
+    desktop-file-utils
+  ] ++ (with rustPlatform; [
+    rust.cargo
+    rust.rustc
+    cargoSetupHook
+  ]);
+
+  buildInputs = [
+    glib
+    gtk4
+    libadwaita
+    ffmpeg-full
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-editing-services
+    gst-plugins-base
+    gst-plugins-good
+    gst-plugins-bad
+    gst-plugins-ugly
+    gst-libav
+  ]);
+
+  preFixup = ''
+    gappsWrapperArgs+=(
+    --prefix PATH : "${lib.makeBinPath [ffmpeg-full]}"
+    )
+  '';
+
+  # delete unused postinstall.py line
+  postPatch = ''
+    sed -i '/postinstall.py/d' meson.build
+  '';
+
+  postInstall = ''
+    glib-compile-schemas $out/share/glib-2.0/schemas
```

Comment:
```suggestion
    patchShebangs build-aux/meson/postinstall.py
```

Reply:
But it would be even better to switch to the new Meson API upstream, see https://gitlab.gnome.org/GNOME/d-spy/-/merge_requests/6/diffs, for an example.

Reply:
What do you mean it can’t work?

Reply:
I can't find in `dconf-editor`.